provisioning/baremetal_secrets.go

```diff
@@ -29,6 +30,7 @@ const (
 	inspectorSecretName      = "metal3-ironic-inspector-password"
 	inspectorUsername        = "inspector-user"
 	tlsSecretName            = "metal3-ironic-tls" // #nosec
+	baremetalJiraComponent   = "Bare Metal Hardware Provisioning / baremetal-operator"
```

Comment:
I think we have a separate component for cluster-baremetal-operator

Reply:
not sure what's the difference between the two, but sure, lets make it cluster-baremetal-operator

Reply:
Two unfortunately named and related but still quite different projects.